.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: main
+on:
+  - push
+  - pull_request
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby-version:
+          - 2.5
+          - 2.6
+          - 2.7
+          - 3.0
+        gemfile:
+          - rails_52.gemfile
+          - rails_60.gemfile
+          - rails_61.gemfile
+        exclude:
+          - ruby-version: [ 2.6, 2.7, 3.0 ]
+            gemfile: rails_52.gemfile
+          - ruby-version: 3.0
+            gemfile: rails_60.gemfile
```

Comment:
FYI: Here is excluded unsupported Ruby versions (incompatible with Rails)

Reply:
:+1: